sleap/gui/dialogs/formbuilder.py

```diff
@@ -458,14 +465,16 @@ def add_item(self, item: Dict[Text, Any]):
             self.addRow(field)
             return
 
-        # double: show spinbox (number w/ up/down controls)
-        elif item["type"] == "double":
-            
-            #If the label of the item is Loss Weight then we will set the min and max to 0 and 1000
-            if item["label"] == "Loss Weight":
+        elif (
+            item["type"] == "double"
+        ):  # double: show spinbox (number w/ up/down controls)
```

Comment:
I liked these better as block comments. Why? 
1. They're easier to see/read
2. The comment explains the block of code to follow not just the single line

```suggestion
        # double: show spinbox (number w/ up/down controls)
        elif item["type"] == "double":
```